### ISSUE TITLE
Cant Start Anything!

### ISSUE THREAD

**Issue by GhostSyndic8** [mtaheij/PoGo-UWP#62]
Device info
------------------
- Phone type: (e.g. Lumia 650)
- App version: (e.g. 1.3.2)

Steps to reproduce
----------------
Start game

Actual behavior
-----------------
Theme starts, stops and then it just keeps loading.

Expected behavior
---------------
Be able to play the game

Other information
---------------

**Comment by GhostSyndic8** [comment]
![wp_ss_20170411_0001](https://cloud.githubusercontent.com/assets/27327357/24942165/3c9acb42-1f1d-11e7-929d-6d2efe9ab924.png)
This is a screenshot of what text I get.

**Comment by ntaheij** [comment]
It means, that you don't have a pokehash key, or it is expired

**Comment by ntaheij** [comment]
So far as I know is that the problem

**Comment by GhostSyndic8** [comment]
I literally just bought it like a couple hours ago

**Comment by ntaheij** [comment]
Ow.. Did you enter it?

**Comment by GhostSyndic8** [comment]
Yeah I entered it, then I entered my PTC info.

**Comment by ntaheij** [comment]
And then you get that error?

**Comment by GhostSyndic8** [comment]
![wp_ss_20170412_0001](https://cloud.githubusercontent.com/assets/27327357/24942649/5df45fc6-1f20-11e7-80dd-4971557b39f8.png) 
I just re installed it and got this error after putting in the key

**Comment by ntaheij** [comment]
There we have a issue at, I wi look with one it is for you

**Comment by GhostSyndic8** [comment]
Okay, anything I can do to get this game started? Just wait it out or ?

**Comment by ntaheij** [comment]
Can you try to go on 4G instead of wifi, and then try again?

**Comment by GhostSyndic8** [comment]
Yeah let me check if anything changes

**Comment by ntaheij** [comment]
#55 is the one with the same error, I got to go school in any moment, so I can't help you now, probably in the afternoon I can hell you again, if this doesn't work please look at #55

**Comment by ntaheij** [comment]
Hell = Help.. I am sorry

**Comment by GhostSyndic8** [comment]
I switch to 4G and I'm still getting the arg null error

**Comment by ntaheij** [comment]
Probably, as I said, you can look at #55, because I got to go now. Have a nice day, if it works :)

**Comment by GhostSyndic8** [comment]
Ok no problem talk to you soon

**Comment by ntaheij** [comment]
I looked for you, but didn't find anything

**Comment by GhostSyndic8** [comment]
Damn thats really sucks, I can't think of anything to do either

**Comment by GhostSyndic8** [comment]
Hey @mtaheij is there a patch or something could possibly correct this?

**Comment by mtaheij** [comment]
I will try tonight, (CET)

**Comment by BadiBasso** [comment]
@GhostSyndic8 Do you have a brand new PTC account? If you have you need to login to PTC and accept an EULA for Pokemon GO in the profile settings.

**Comment by GhostSyndic8** [comment]
@BadiBasso no I don't have a new one, been using the same PTC account since @mtaheij took over Pogo. I will still try and see if anything changed on my account

**Comment by GhostSyndic8** [comment]
Any progress on this? @mtaheij

**Comment by mtaheij** [comment]
Nothing further than that calling the hashing service sometimes returns an invalid result. The API we are using is not equipped well enough to handle this correctly.
Still investigating...

**Comment by GhostSyndic8** [comment]
Hey @mtaheij I've recently updated the game to v 1.3.4 and it keeps giving me an error saying that something went wrong asking me to sign out constantly.

**Comment by GhostSyndic8** [comment]
![wp_ss_20170424_0002](https://cloud.githubusercontent.com/assets/27327357/25348621/cd23fa48-28ec-11e7-81df-bb9fe27cb481.png)
![wp_ss_20170424_0001](https://cloud.githubusercontent.com/assets/27327357/25348622/cd2795fe-28ec-11e7-90a1-71b0b006c301.png)

**Comment by GhostSyndic8** [comment]
The first screen shot is what happens when I press no

**Comment by mtaheij** [comment]
Can you try setting your phone to English (US-en), restart your phone and try again?

**Comment by GhostSyndic8** [comment]
It was always set to English but I checked again just in case. I restarted my phone and it's still doing the same thing 😩

**Comment by GhostSyndic8** [comment]
![wp_ss_20170425_0003](https://cloud.githubusercontent.com/assets/27327357/25389498/cf18f012-299e-11e7-8618-f8b10be74518.png)
![wp_ss_20170425_0002](https://cloud.githubusercontent.com/assets/27327357/25389495/cf059e9a-299e-11e7-9335-502908e560e8.png)
![wp_ss_20170425_0001](https://cloud.githubusercontent.com/assets/27327357/25389496/cf06d4f4-299e-11e7-9394-0a11d6905b97.png)


![wp_ss_20170425_0004](https://cloud.githubusercontent.com/assets/27327357/25389497/cf0a828e-299e-11e7-81ca-a2442fbd7982.png)

**Comment by BadiBasso** [comment]
@GhostSyndic8 Could reproduce the same errors when I logged in with a banned account. All inventories empty, no player stats, no pokedex entries.
@mtaheij Current code does not correctly handle Niantic http response code 3 (bad_request) which most of the time means a banned account.

**Comment by GhostSyndic8** [comment]
@BadiBasso I used it today on my girlfriends iPhone and everything was running normal.

**Comment by BadiBasso** [comment]
@GhostSyndic8 Good to hear that your acc is not banned despite the symptoms :)
There's not much more I can do. Have to wait if HockeyApp logs could provide some more info.

**Comment by GhostSyndic8** [comment]
@BadiBasso yeah I even emailed niantic saying I had issues on an iPhone, but they assured the account wasn't banned, it wasn't even flagged for anything, but thanks for the reply :)

**Comment by ashwshukla** [comment]
@mtaheij now the minimal Api version of ur UWP is 0.73.1 and we are runnig hasher version 0.67.1
That's the reason api isn't working so please make a new update or a bypass to skip that pop up displayed while playing the game.
